extension/content.css

```diff
@@ -478,3 +478,8 @@ hidden content area created to increase hover target
 	padding-top: 0 !important;
 	border-top: none !important;
 }
+
+.comment-full-name {
+	font-weight: normal;
+	margin-left: 1%;
```

Comment:
No problem - no need for % anyway - will update with the rest of the changes tonight